### DISCUSSION TITLE
ログイン処理の認可コードについて

### DISCUSSION THREAD

**Discussion by TwilightWings** [pixiv/vroid-sdk-developers#9] [Q&A] [answered]
https://developer.vroid.com/sdk/#get-started

サンプルでは、「認可コードを手動入力する場合の処理」というサンプルが示されておりますが、
手動入力をさせない方法があるのでしょうか？

ログイン処理をアプリ内で完結し、認可コード取得をSDK内部に隠蔽させる方法があればご教示いただきたいです。

**Comment by Haresaku** [comment, marked as answer]
現在、手動入力以外の対応方法はiOS/Android環境でのURLスキームを使ったもののみサポートしております。
Desktop環境の場合は手動入力で対応お願いします。

https://github.com/pixiv/vroid-sdk-developers/blob/main/README.ja.md#oauth%E3%81%AE%E8%AA%8D%E8%A8%BC

**Reply by TwilightWings** [reply]
ご回答ありがとうございます。現在Desktop環境の開発ですので手動入力で対応いたします。
